fake/file.go

```diff
@@ -0,0 +1,36 @@
+package fake
+
+import (
+	"os"
+	"path/filepath"
+
+	check "gopkg.in/check.v1"
+)
+
+func TempDirectory(parentDirectory string, c *check.C) string {
+	if parentDirectory != "" {
+		_, err := os.Stat(parentDirectory)
+		c.Assert(err, check.IsNil)
+	}
+
+	tempDir, err := os.MkdirTemp(parentDirectory, "test-")
+	c.Assert(err, check.IsNil)
+	return tempDir
+}
+
+func TempFile(directory, fileName, content string, c *check.C) *os.File {
```

Comment:
Can we use verb + noun as a function name like `MakeTempDirectory` and `MakeTempFile`? Using a noun as a name might make the function unclear.

Reply:
Added a section in the [Wiki](https://github.com/longhorn/longhorn/wiki/Coding-Convention#functions).